src/app/globals.css

```diff
@@ -1,7 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
+@import 'bulma/css/bulma.css';
```

Comment:
Better to use customized bulma.scss from old code instead of importing the lib directly.